CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-[...]
+- **[FIX]** In `Itinerary` Only supports `withTime` for `places`
```

Comment:
nit: `- **[FIX]** Always display time in `Itinerary` with segments

Reply:
Much better, thanks